analysis/config/config.go

```
@@ -403,6 +403,12 @@ func LoadFromFiles(configFileName string) (*Config, error) {
 //
 //gocyclo:ignore
 func Load(filename string, configBytes []byte) (*Config, error) {
+	// Get absolute path to config. Other files in config will be relative to where the config is.
+	pathToConfig := filename
+	if !path.IsAbs(filename) {
+		wd, _ := os.Getwd()
```

Should probably return the error